models/board.js

```diff
@@ -0,0 +1,41 @@
+const { Model, DataTypes } = require('sequelize');
+const sequelize = require('../config/connection');
+const User = require('./User');
+
+// create Board model
+class Board extends Model { }
+
+// define table columns and configuration
+Board.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    // board_content: {
+    //   type: DataTypes.JSON,
+    //   allowNull: false,
+    // },
```

Comment:
Yes, this is where the Fabric JSON for the board would be pulled. Since we don't have that format yet, I just left it in and commented it out as we will need it later, but it is not yet defined.

Reply:
Oh, well in the future you can just comment this out and remove teh allowNull line. Then it will create the column. Pretty sure this is going to be either json or string... to make thing easier I would just make it a string. In any case, I don't think this should block the PR so I am resolving this. We will get it settled when we can save the canvas.